app/classes/query/modules/titles.rb

```diff
@@ -9,9 +9,12 @@ module Titles
 
       def initialize_title
         @title_tag = "query_title_#{flavor}".to_sym
-        @title_args = { type: model.to_s.underscore.to_sym }
+        @title_args = { type: model.to_s.underscore.
+                      tr("/", "_"). # for namespaced models
+                      to_sym }
```

Comment:
This would create an ambiguity if we ever had models `A::B` and `A_B`.
But I think that's very unlikely, and don't want to spend the time to engage in Speculative Generality.

Reply:
Agreed.

Reply:
Late to the game, but looks good to me. :)